platform/ios/src/NSOrthography+MGLAdditions.m

```diff
@@ -3,6 +3,10 @@
 @implementation NSOrthography (MGLAdditions)
 
 + (NSString *)mgl_dominantScriptForMapboxStreetsLanguage:(NSString *)language {
+    if (language == nil) {
```

Comment:
If `language` is `nil`, the reason is that `+[MGLVectorTileSource preferredMapboxStreetsLanguage]` returned `nil`. `-[MGLFeatureAccessibilityElement initWithAccessibilityContainer:feature:]` should use the `name` attribute instead of trying and failing to use the `name_(null)` attribute:

https://github.com/mapbox/mapbox-gl-native-ios/blob/ddf8e35970d1a8a46171bd798bcfa8f22d2760e0/platform/ios/src/MGLMapAccessibilityElement.mm#L51-L52

which would result in no accessibility label at all, assuming that `name` is present on the feature:

https://github.com/mapbox/mapbox-gl-native-ios/blob/ddf8e35970d1a8a46171bd798bcfa8f22d2760e0/platform/ios/src/MGLMapAccessibilityElement.mm#L70

Reply:
When languageCode is null, nameAttribute ends up being `name_(null)`, and on line 53, name is nil. When name is nil, the fallback on line 56/57 is still applied.

I can still (and almost did initially) update this to be more defensive about `nil` `languageCode` though

Reply:
Updated

Reply:
Yeah, I think that’s necessary because `name` is almost always present, so that fallback would rarely be applied.

Reply:
Never mind, I misread this as a `== nil` rather than a `!= nil`:

https://github.com/mapbox/mapbox-gl-native-ios/blob/ddf8e35970d1a8a46171bd798bcfa8f22d2760e0/platform/ios/src/MGLMapAccessibilityElement.mm#L56